objects/src/transaction/inputs.rs

```diff
@@ -215,17 +210,23 @@ impl<T: ToNullifier> InputNotes<T> {
             }
         }
 
-        let commitment = build_input_notes_commitment(&notes);
+        let nullifier_commitment = nullifier_commitment(&notes);
 
-        Ok(Self { notes, commitment })
+        Ok(Self { notes, nullifier_commitment })
     }
 
     // PUBLIC ACCESSORS
     // --------------------------------------------------------------------------------------------
 
-    /// Returns a commitment to these input notes.
-    pub fn commitment(&self) -> Digest {
-        self.commitment
+    /// Returns sequential hash of all note's nullifiers.
+    ///
+    /// For non empty lists the commitment is defined as:
+    ///
+    /// > hash(nullifier_0 || ZERO || nullifier_1 || ZERO || .. || nullifier_n || ZERO)
+    ///
+    /// Otherwise defined as ZERO for empty lists.
+    pub fn nullifier_commitment(&self) -> Digest {
```

Comment:
At first I thought the commitment was over the note data, this should clarify it.